### DISCUSSION TITLE
Radar

### DISCUSSION THREAD

**Discussion by Blue2hark** [flybywiresim/aircraft#6260] [General]
Hello I think we have all noticed that the 32nx does not have terrain and weather radar which can be a bit annoying I would like to know if FBW can install the Asobo radar we will not wait 1 year to have a radar it is early at night and in addition a very complicated weather forecast with snow so we flew blind. Sad.sorry for the poor english

**Comment by CptBayer** [comment]
Hi, isn´t it implemented in the stable version? I only fly dev version ...

greets

**Reply by Blue2hark** [reply]
I only use the dev version except you noticed that there hasn't been a radar for a week and the same will be true for the stable version in the next few months and I find it a shame to do the LOWG airport without radar land with clouds at 200ft you can hang on my little one MDR

**Reply by Blue2hark** [reply]
FBW could at least wait until you get access to the SDK, you don't remove the radar without having access to it. as if, for example, in the restoration you have problems you do not throw it away you are waiting for new material.after it is my vision of logic I work in logic not in haste

**Reply by tracernz** [reply]
We prioritised an accurate ND, flight planning and guidance system over the terrain and weather display as it wasn't possible to have both. There is work ongoing for a terrain display but we cannot do anything about weather until the sim has a suitable API.

**Reply by regis-40** [reply]
supprimé le radar météo qui fonctionné ??????????? il serait plus judicieux de réinstaller le radar surtout pour les franchissement de turbulence surtout pour contourner la zone.

**Reply by Blue2hark** [reply]
> 
it would be nice if users reinstall the radar vote on Twitter discord you will see the whole community say yes to the radar where you are based only on your personality and not the user.the SDK does not have access to the radar so do not penalize the user please

**Reply by CptBayer** [reply]
wow blue2hark,


don´t forget that this is freeware !!!!! Development is develpment, when it´s finished it´s finished .... be a bit more thankfull mate ... or fly the crap from asobo with radar .... 

greets

**Reply by Blue2hark** [reply]
you if you like to take the mountain do you know that we have a problem you flew to europe in austria for example with all the surrounding mountains with clouds of snow as for the moment impossible to do without radar after if you didn't want to understand.I should have kept the file I'm stupid how I should have known that FBW during the day by taking him off the radar without warning you see the thing too

**Reply by CptBayer** [reply]
sorry blueshark, I am using Navigraph and I also had a lot of approaches into LOWI. I know the surroundings quite well. So for me it´s no problem. I understand your arguments, but I think that is not a reason to complain. All the guys spending a lot of freetime for a really fantastic product till now..... so we all need some patience.... CHAKKA :-)

**Reply by tracernz** [reply]
You are also not showed to use the terrain display (it’s not a radar, but a pre-loaded database) for primary means of terrain avoidance IRL. That’s what instrument procedures and VMC are for.

**Comment by regis-40** [comment]
ok merci pour la réponse
________________________________
De : Michael Corcoran ***@***.***>
Envoyé : mardi 30 novembre 2021 22:43
À : flybywiresim/a32nx ***@***.***>
Cc : regis-40 ***@***.***>; Comment ***@***.***>
Objet : Re: [flybywiresim/a32nx] Radar (Discussion #6260)


You are also not showed to use the terrain display (it’s not a radar, but a pre-loaded database) for primary means of terrain avoidance IRL. That’s what instrument procedures and VMC are for.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/flybywiresim/a32nx/discussions/6260#discussioncomment-1724930>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ARENTUVNJGQJHCZNSXIX263UOVAPDANCNFSM5I5B5FYA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
